.changeset/cool-ads-carry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@jpmorganchase/mosaic-plugins': minor
+---
+
+Add a `filter` option to SearchIndexPlugin, which allows you to exclude pages from the search index.
```

Comment:
the plugin  already has include in its naming convention, should we call this excludePages (or similar) rather than filter